<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>1148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="open_button">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="song_info_layout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="song_cover">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../Resources/resources.qrc">:/Icons/Album.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="song_name_label">
          <property name="text">
           <string>Song Name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="song_album_label">
          <property name="text">
           <string>Song Album</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="song_quality_label">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="song_controls_layout">
      <item>
       <widget class="QLabel" name="song_mini_cover">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../Resources/resources.qrc">:/Icons/Album.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="song_name_controls_label">
        <property name="text">
         <string>Song Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="previous_button">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../Resources/resources.qrc">
          <normaloff>:/Icons/Previous.png</normaloff>:/Icons/Previous.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="play_button">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../Resources/resources.qrc">
          <normaloff>:/Icons/Play.png</normaloff>:/Icons/Play.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="next_button">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../Resources/resources.qrc">
          <normaloff>:/Icons/Next.png</normaloff>:/Icons/Next.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="mute_button">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../Resources/resources.qrc">
          <normaloff>:/Icons/Mute.png</normaloff>:/Icons/Mute.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="volume_slider">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="song_length_label">
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="song_progress">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="song_progress_label">
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
